src/junitParser.ts

```diff
@@ -60,8 +60,7 @@ interface JUnitSuite {
     testcase: JUnitTestCase | JUnitTestCase[];
 }
 interface JUnitReport {
-    testsuite?: JUnitSuite;
-    testsuites?: { testsuite?: JUnitSuite };
+    testsuites?: { testsuite?: JUnitSuite | JUnitSuite[] };
```

Comment:
We should still keep `testsuite?: JUnitSuite;` to support other `junit` files as well (checkout the smtp-server)